libs/types/src/cdf/cast-vote-records/index.ts

```diff
@@ -1468,6 +1468,11 @@ export interface ImageData {
    * A pointer to the location of the image file.
    */
   readonly Location?: Uri;
+
+  /**
+   * The hash of the VotingWorks layout file corresponding to this image.
+   */
+  readonly vxLayoutFileHash?: string;
```

Comment:
The CDF CVR schema includes fields for image hashes, which is great! For layout file hashes, I've opted for a custom field on CDF `CVR.ImageData` objects.

I could've alternatively created a separate file for those hashes, but that would've complicated upstream code.